homeassistant/components/zwave.py

```diff
@@ -88,6 +95,32 @@ def _obj_to_dict(obj):
             if key[0] != '_' and not hasattr(getattr(obj, key), '__call__')}
 
 
+def _node_name(node):
+    """ Returns the name of the node. """
+    return node.name or "{} {}".format(
+        node.manufacturer_name, node.product_name)
+
+
+def _value_name(value):
+    """ Returns the name of the value. """
+    return "{} {}".format(_node_name(value.node), value.label)
+
+
+def _object_id(value):
```

Comment:
Did you really have to move these functions outside of the class? 
Couldn't you just forward the customization config into the class and class extract the config values inside the class?